packages/project-utils/bundling/function/buildFunction.js

```diff
@@ -54,4 +73,21 @@ module.exports = options => {
             resolve();
         });
     });
+
+    const handlerFile = fs.readFileSync(options.cwd + "/build/handler.js", {
+        encoding: "utf8",
+        flag: "r"
+    });
+    const includesGraphQl = handlerFile.includes("handler-graphql");
```

Comment:
How does this perform BTW? Is it an intensive operation / an operation that takes a lot of time to execute? Just wondering...

Reply:
this bundle was on the smaller size but its pretty quick at 10ms on average